test/helpers/index.ts

```diff
@@ -37,11 +38,14 @@ interface ContractInstances {
   certificate: Certificate;
   fifoMarket: FIFOMarket;
   lNori: LockedNORI;
+  rNori: RestrictedNORI;
   removalTestHarness: RemovalTestHarness;
   mockCertificate: MockCertificate; // todo key remapping of Contracts
   mockERC1155PresetPausableNonTransferrable: MockERC1155PresetPausableNonTransferrable; // todo consider TestHarness vs Mock naming convention
 }
 
+export const NOW = Math.floor(Date.now() / 1000);
```

Comment:
it's not a great idea to rely on this FYI-- if possible, rely on `getLatestBlockTime` instead

Reply:
do you think this still matters if I'm basically just using `NOW` as a way to generate an arbitrary but valid timestamp to use as schedule start times?  I guess I am kinda relying on it in the sense that unless I explicitly `advanceTime`, I'm assuming that we're in a state where none of the schedule window has transpired yet so it's important that `NOW` and the latest block time are roughly the same.  will see about replacing

Reply:
Should be a fairly easy replacement. The NOW pattern created some flakiness in early iterations of the lNORI test suites due to it being evaluated and assigned independently of the actual blockchain time

Reply:
is this still used?